website/docs/language/state/remote-state-data.mdx

```diff
@@ -9,17 +9,12 @@ description: >-
 
 [backends]: /language/settings/backends
 
-The `terraform_remote_state` data source retrieves the root module output values
-from some other Terraform configuration, using the latest state snapshot from
-the remote backend.
+The `terraform_remote_state` data source uses the latest state snapshot from the remote backend to retrieve the root module output values
+from some other Terraform configuration.
```

Comment:
```suggestion
The `terraform_remote_state` data source uses the latest state snapshot from a specified state backend to retrieve the root module output values
from some other Terraform configuration.
```

^^ The arguments for the data source let you access pretty much any state backend -- like, you could reach out and grab some state from S3 during a run in terraform cloud.